tests/fixture/environment.go

```diff
@@ -28,7 +23,6 @@ func NewTestEnvironment(
 ) *e2e.TestEnvironment {
 	// Run only once in the first ginkgo process
 	nodes := tmpnet.NewNodesOrPanic(flagVars.NodeCount())
-	nodes[0].Flags[config.HTTPPortKey] = config.DefaultHTTPPort
```

Comment:
This is intended behavior https://github.com/ava-labs/hypersdk/pull/1320.

We should continue to support a stable network identity at least via the run script